github/client.go

```diff
@@ -274,7 +274,10 @@ type ReleaseAsset struct {
 	ApiUrl      string `json:"url"`
 }
 
-func (client *Client) FetchReleases(project *Project) (releases []Release, err error) {
+// FetchReleases fetches Releases from project, optionally including
+// drafts or excluding prereleases. If limit is positive, it returns at
+// most limit Releases.
+func (client *Client) FetchReleases(project *Project, includeDrafts, excludePrereleases bool, limit int) (releases []Release, err error) {
```

Comment:
Maybe it would be cleaner to be consistent here and make that parameter `includePrereleases`?